site/_resources/upgrading.md

```diff
@@ -5,126 +5,122 @@ layout: page
 
 This document describes the changes needed to upgrade your Contour installation.
 
-## Upgrading Contour 1.0.0-rc.1 to 1.0.0-rc.2
+# Upgrading Contour 0.15.3 to 1.0.0
 
-Contour 1.0.0-rc.2 is the second release candidate for Contour 1.0.
+Contour 1.0.0 is the current stable release.
 
-### Release candidate
+<div class="alert-deprecation">
+<b>Deprecation Notice</b></br>
+<p>
+The <code>IngressRoute</code> CRD has been deprecated and will not receive
+further updates. Contour 1.0.0 continues to support the IngressRoute
+API, however we anticipate it will be removed in the future.
+<p>
+</p>
+Please see the documentation for <a href="/docs/1.0/httpproxy"><code>HTTPProxy</code></a>,
+which is the successor to <code>IngressRoute</code>.
+You can also read the <a href="/guides/ingressroute-to-httpproxy">IngressRoute to HTTPProxy upgrade</a>
```

Comment:
same, please use the {% link syntax to help catch broken links.

Reply:
Fixed,